src/components/SlideshowHero/_slideshow-hero.scss

```diff
@@ -70,19 +70,20 @@
   }
 }
 
-.slideshow-hero__content {
+.slideshow-hero__copy {
+  font-size: calc(18/16 * var(--text-base-size));
```

Comment:
why the 18/16?

Reply:
18 pixels / the base pixels (in other words the calculation needed to get the decimal to multiply the text-base-size ... that wriggled and jiggled and tickled inside her)

Reply:
nice - seems like this could be a var from `_typography`, but I can also see that the design system declares two variants which are the same size (larger & large)... 😞

might be worth addressing this in the typography settings scss, and then we could have:

```css
--text-larger: calc(20/16 * var(--text-base-size));
--text-large: calc(18/16 * var(--text-base-size));
```

or something to that effect? seems redundant having larger and large the same size